autoload/tagtracks.vim

```diff
@@ -72,6 +80,7 @@ function tagtracks#StartTagTracks()
   " window id whose tagstack we want to display
   const tagtracks_id = win_getid()
   new
+  set nonumber
```

Comment:
```suggestion
  setlocal nonumber
```

Let's not disable the user's `number` globally :)

Reply:
Interestingly, only the tag-tracks window's line numbers were disabled with `set`. My code window still had its line numbers. vim 8.2 on macOS, if it matters. Changed to `setlocal` though.

Reply:
Ah, I see now that `'number'` is local to a window. The reason it works to `:set number` in a vimrc is because window-options are inherited when splitting. Anyway, `:setlocal` is clearer here, so let's keep it.